src/back/msl.rs

```diff
@@ -924,8 +907,56 @@ impl<W: Write> Writer<W> {
             )?;
 
             let fun_name = fun.name.or_index(fun_handle);
+            let result_type_handle = fun.return_type.unwrap();
+            let result_type_name = module.types[result_type_handle]
+                .name
+                .or_index(result_type_handle);
+            writeln!(self.out, "{} {}(", result_type_name, fun_name)?;
+
+            for (index, &ty) in fun.parameter_types.iter().enumerate() {
+                let name = Name::from(ParameterIndex(index));
+                let member_type_name = module.types[ty].name.or_index(ty);
+                let separator = separate(index + 1 == fun.parameter_types.len());
+                writeln!(self.out, "\t{} {}{}", member_type_name, name, separator)?;
+            }
+            writeln!(self.out, ") {{")?;
+
+            // write down function body
```

Comment:
Nit: I think this is a leftover comment from down below, as I have seen it at line [1137](https://github.com/gfx-rs/naga/pull/200/files#diff-aaa1f30b3e995f65d9bd7557d98981fdR1137).